libflux/go/libflux/flux.h

```diff
@@ -75,6 +75,10 @@ void flux_get_env_stdlib(struct flux_buffer_t *);
 // flux_ast_pkg_t* argument, so it should not be reused after calling this function.
 struct flux_error_t *flux_analyze(struct flux_ast_pkg_t *, struct flux_semantic_pkg_t **);
 
+// flux_ast_pkg_t merges the files of a given input AST package into the file vector of a
+// given output AST package. If there are errors, then a panic will occur.
+struct flux_error_t *flux_merge_ast_pkg_files(struct flux_ast_pkg_t *, struct flux_ast_pkg_t *);
```

Comment:
We should name the args to this function since its not clear which is the input and which is the output without names.